### PR TITLE
KK-133 | implement 404 page follow new design

### PR DESCRIPTION
**Pull request by midori0507**

### CODE REVIEW

File: src/common/translation/i18n/fi.json [hunk 1]
```diff
@@ -387,5 +394,6 @@
     "postalCode": {
       "invalidFormat": "Väärä formaatti"
     }
-  }
+  },
+  "#REF!": "#REF!"
```
Comment (midori0507) [inline src/common/translation/i18n/fi.json]: Apparently @badrange is the only one know how to fix this annoying thing

File: src/common/translation/i18n/fi.json [hunk 2]
```diff
@@ -226,7 +230,10 @@
     }
   },
   "notFound": {
-    "text": "Etsimääsi sivua ei löytynyt"
+    "text": "Etsimääsi sivua ei löytynyt",
+    "return": {
+      "text": "Return to service"
```
Comment (jimppa) [inline src/common/translation/i18n/fi.json]: Shouldn't this be in finnish?

File: src/common/translation/i18n/sv.json [hunk 3]
```diff
@@ -226,7 +230,10 @@
     }
   },
   "notFound": {
-    "text": "Sidan du letade efter kunde inte hittas"
+    "text": "Sidan du letade efter kunde inte hittas",
+    "return": {
+      "text": "Return to service"
```
Comment (jimppa) [inline src/common/translation/i18n/sv.json]: Shouldn't this be in swedish?

File: src/domain/app/notFound/NotFound.tsx [hunk 4]
```diff
@@ -1,15 +1,26 @@
...
+  const history = useHistory();
   return (
     <Container>
-      <div className="not-found">
+      <div className={styles.notFound}>
+        <h2>404</h2>
+        <Icon src={adultFace} className={styles.icon} alt="Sad adult face" />
         <p>{t('notFound.text')}</p>
+
+        <Button onClick={() => history.push('/')}>
```
Comment (jimppa) [inline src/domain/app/notFound/NotFound.tsx]: This should probably be a `<Link>`

Comment (midori0507) [reply src/domain/app/notFound/NotFound.tsx]: I think it should be Button with function of a link, as the design is a button :D

Comment (jimppa) [reply src/domain/app/notFound/NotFound.tsx]: No.

Comment (midori0507) [reply src/domain/app/notFound/NotFound.tsx]: Screenshot 2020-01-15 at 13 22 45

Comment (jimppa) [reply src/domain/app/notFound/NotFound.tsx]: You can style an anchor-element to look like a button. It's a link, not a button.

Comment (midori0507) [reply src/domain/app/notFound/NotFound.tsx]: okay :sad:

**Pull request merged**